test/features/pagecontent/save_api.js

```diff
@@ -110,6 +111,23 @@ describe('page save api', function() {
     });
 
     it('fail for bad token', function() {
+        var apiURI = server.config
+            .conf.templates['wmf-sys-1.0.0']
+            .paths['/{module:action}']['x-modules'][0].options.apiRequest.uri;
+        apiURI = apiURI.replace('{domain}', 'en.wikipedia.beta.wmflabs.org');
+
+        nock.enableNetConnect();
+        var api = nock(apiURI)
+            // The first request should return a page so that we store it.
```

Comment:
Indentation. Also, the comment doesn't seem to be in line with what it returns.

The same comment applies to all nock additions in this PR.